src/cardStyles/stars.css

```diff
@@ -0,0 +1,19 @@
+.star {
+  display: flex;
+}
+
+.star__stars {
+  width: 16px;
+  height: 16px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.card__star :nth-child(-n+4) {

```

Comment:
it should work not only for 4 starts 

<img width="833" alt="Screenshot 2023-03-11 at 18 00 55" src="https://user-images.githubusercontent.com/32853034/224499594-21400124-deda-4ebe-920e-405bb3215281.png">


also apply this styles for `.star__stars`, not .`card__start`